src/Repository/UsersRepository.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Users;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+/**
+ * @method Users|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Users|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Users[]    findAll()
+ * @method Users[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class UsersRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, Users::class);
+    }
+
+    /*
```

Comment:
jw

Reply:
Powiem Ci, że nie wiem skąd to tutaj się wzięło. Jak wygenerowałem plik, to nic w nim nie zmieniałem,także PHP musiał coś źle zrobić.